vnext/Microsoft.ReactNative/Modules/TimingModule.h

```diff
@@ -3,96 +3,116 @@
 
 #pragma once
 
-#include <CppWinRTIncludes.h>
+#include <InstanceManager.h>
 #include <cxxreact/CxxModule.h>
 #include <cxxreact/MessageQueueThread.h>
 
-#include <folly/dynamic.h>
+#include <chrono>
 #include <memory>
 #include <vector>
 
-#include <winrt/Windows.Foundation.h>
-namespace Microsoft::ReactNative {
+#include <windows.h>
 
-typedef winrt::Windows::Foundation::DateTime TDateTime;
-typedef winrt::Windows::Foundation::TimeSpan TTimeSpan;
+namespace facebook {
+namespace react {
```

Comment:
why is this in the facebook::react namespace?

Reply:
Its currently being exported from the desktop dll, so I didn't want to change the name.  But it doesn't look like we are using it anymore, so I'll put it in the MS.RN namespace, and remove it from the def files.